GameData/RealismOverhaul/Waterfall_Configs/CryoEngines/CryoEnginesWF.cfg

```diff
@@ -0,0 +1,365 @@
+//=============
+//	Waterfall
+//=============
+@PART[cryoengine-deinonychus-1]:BEFORE[ROWaterfall]:NEEDS[Waterfall]
+{
+	//kill the stock smoke plumes
+	@EFFECTS
+	{
+		@fx-deinonychus
+		{
+			!PREFAB_PARTICLE {}
+		}
+	}
+
+	//just rescale the plumes
+	@MODULE[ModuleWaterfallFX]
+	{
+		@TEMPLATE
+		{
+			@scale[*] *= #$/rescaleFactor$
+		}
+	}
```

Comment:
This gets to be *a lot* of repetition in the file.  Consider instead just one generic patch for the parts that use it?
```
@PART[a,b,c,d,e]:BEFORE[ROWaterfall]
{
	@MODULE[ModuleWaterfallFX] { @TEMPLATE { @scale[*] *= #$/rescaleFactor$ } }
}
```

Reply:
I have changed that. I hope it's correct now)